### ISSUE TITLE
Count array elements based on property filter returns unexpected count value

### ISSUE THREAD

**Issue by surfmuggle** [rangav/thunder-client-support#874]
In my testcase i want to test if an array only contains 1 element with a certain name ` json.products[name=Cranberries] count 1`

- Expected: count = 1 should pass because only 1 product has the name Cranberries
- Actual: fail count returns 2 despite that there is only 1 product named Cranberries

Calling `api.predic8.de/shop/products/` like this

```
curl -X GET \
  'https://api.predic8.de/shop/products/' \
  --header 'Accept: */*' \
```

returns a response close to this one

```
{
  "meta": { "count": 32, "limit": 10, "page": 1,
    "next_url": "/shop/products/?page=2&limit=10"
  },
  "products": [
    {  "name": "Oranges", "product_url": "/shop/products/10" },
    ... 
    {  "name": "Cranberries", "product_url": "/shop/products/57" },
    {  "name": "Raspberries", "product_url": "/shop/products/90" },
    {  "name": "Cherries", "product_url": "/shop/products/7" },
    {  "name": "Apple", "product_url": "/shop/products/18" },
    {  "name": "Apples", "product_url": "/shop/products/50" }
  ]
}
```

As you can see there is only 1 product named `Cranberries` and one named `Oranges`

```
// fails - but should pass
Query: json.products[name=Cranberries] property count is 1 => Actual: count is 2

// pass - but should fail
Query: json.products[name=Cranberries] property count is 2 
```

Same problem with oranges

![image](https://user-images.githubusercontent.com/1271555/201482402-2cf6d0e6-bcf8-4d28-9c53-53a65958a057.png)

**Comment by rangav** [comment]
@surfmuggle 

this is echo endpoint, what ever you post in body will return the same as response - useful for testing.
https://www.thunderclient.com/t/echo

**Comment by rangav** [comment]
The array filter `products[name=Cranberries]` and count will not work currently, will look into it.

**Comment by rangav** [comment]
This feature is now implemented and published to marketplace, please update to `v2.0.0`

See all features released
https://github.com/rangav/thunder-client-support/releases/tag/v2.0.0

Please let me know your feedback.
